src/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+  height: 70px;
+}
+
+.search-input-label {
+  position: relative;
+}
+
+.search-form-small {
+  height: 42px;
+}
+
+.search-input {
+  width: 100%;
+  height: 100%;
+  padding: 0 0 0 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 900;
+}
+
+.search-input-label:after {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 25px;
+  content: "";
+  background: url(./images/Search.svg) no-repeat;
+  display: block;
+  width: 19px;
+  height: 19px;
+  background-size: cover;
+}
+
+.search-input-label-small::after {
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
+
+.search-input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
```

Comment:
you should put this styles under `.search-input` selector, `...::placeholder` will inherit these styles